### DISCUSSION TITLE
Welcome to Aether Discussions!

### DISCUSSION THREAD

**Discussion by NoiSek** [NoiSek/Aether#94] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by thealmightychuck** [comment]
So I'd like to use this theme, but it sounds like there are some issues with hard-coded folders, I've been trying to set this up but I think I'm missing something, what folders do I need to rename?

**Reply by NoiSek** [reply]
A technical limitation of the medium means that you need to have it named `lightdm-webkit-theme-aether`:

https://github.com/NoiSek/Aether/blob/master/src/js/Logic/FileOperations.js#L16
